src/main/resources/uischema/publishMessages.json

```diff
@@ -0,0 +1,62 @@
+{
+  "connectorName": "kafkaTransport",
+  "operationName": "publishMessages",
+  "title": "Publish Messages",
+  "help": "<h1>Publish Messages</h1> <b>The publishMessages operation allows you to publish messages to the Kafka brokers via Kafka topics.</b><br><br><ul><li><a href=\"https://ei.docs.wso2.com/en/latest/micro-integrator/references/connectors/kafka-connector/kafka-connector-config/\"> More Help </a></li></ul>",
```

Comment:
Can help be a HTML?

Reply:
Shall we check with a new tooling pack and see how this appears?

Reply:
This is not yet supported in Integration Studio. Opened issue: https://github.com/wso2/devstudio-tooling-ei/issues/1148